notes/20200217_meeting.md

```diff
@@ -39,6 +39,9 @@ Other benefits of RAP include:
 ### Increased trust:
 - code and documentation open to anyone (if possible)
 - software open to anyone?
+- know who has edited code, when, what changed and why
+- unit tests and peer review provide reassurance that the code works as expected
```

Comment:
Agreed, although specifying unit tests assumes the code is written in functions - I don't think we want to be that strict for an MVP

Reply:
True, and a fair point. Happy to remove that bullet